### ISSUE TITLE
Custom _Sidebar.md and Relative Links

### ISSUE THREAD

**Issue by dd86k** [gitbucket/gitbucket#2629]
Didn't find a similar Issue so making one.

- Version: 4.35.3 (and pages plugin to 1.9.0 if that's related)
- Deployment: jdk-headless (1.8.0_275) behind nginx

Consider this example for `_Sidebar.md`:
```markdown
# Home
- [FAQ](FAQ)
```

While both Github and Gitlab has the desired effect of proceeding to the `FAQ.md` page, Gitbucket attempts to reach to `https://example.org/user/repo/wiki/_blob/FAQ`, which does not exist, while `https://example.org/user/repo/wiki/FAQ` exists. The normal list of pages still works as-is, but the custom sidebar links adds `_blob`. Haven't tested images yet.

I suspect this is in the markdown parser as it acts the same in a wiki context as for a README.md in the repo files.

Yes, using absolute links fixes it, but I'm pushing the same repo over to different (but compatible) servers.

**Comment by takezoe** [comment]
The following notation is recommended to link to a Wiki page.
```
- [[FAQ]]
- [[Frequently Asked Questions|FAQ]]
```

**Comment by dd86k** [comment]
Ah, interesting. Never seen the latter since I'm used to the former (because Markdown). MD still works fine on Github and Gitlab, tough.

I'll try the [MediaWiki syntax](https://docs.github.com/en/communities/documenting-your-project-with-wikis/editing-wiki-content#adding-links) (now that I've looked this up) and come back to you.

**Comment by dd86k** [comment]
Yep.. Works with MediaWiki style links. I'll keep this open if you want to do something with the Markdown style links in Gitbucket for Wiki sidebars.

**Comment by takezoe** [comment]
For now, I think we have nothing to do. The current link behavior is expected.
